custom_components/tuya_local/devices/tontine_039-WIFI_blanket.yaml

```diff
@@ -0,0 +1,48 @@
+#ConforTech Platinum Electric Blanket
+#Model No:TEB2045QB
+#Controller Unit No: 039-WIFI
+
+name: Tontine Electric Blanket
+products:
+  - id: m3kmcn9evognklsp
+primary_entity:
+  entity: switch
+  icon: "mdi:bed"
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+secondary_entities:
+  - entity: select
+    name: Temperature
+    icon: "mdi:thermometer"
+    dps:
+      - id: 4
+        type: string
+        name: option
+        mapping:
+          - dps_val: low
+            value: "low"
+          - dps_val: middle
+            value: "middle"
+          - dps_val: high
+            value: "high"
+  - entity: number
+    name: Timer
+    icon: "mdi:timer"
+    dps:
+      - id: 9
+        type: integer
+        name: value
+        unit: h
+        range:
+          min: 1
+          max: 10
+  - entity: sensor
+    name: Countdown
+    icon: "mdi:timer"
```

Comment:
Add `class: duration`, it may give a better display.

Reply:
If you want it displayed that way that's fine, although it shows seconds which is always 00, and personally I feel it looks better as minutes, which is how the Tuya app displays it.

Reply:
You are probably right looking at them side by side, I think more people would be happier with duration. I just updated it.